src/Tribe/Views/V2/Views/Traits/List_Behavior.php

```diff
@@ -91,11 +91,21 @@ static function ( $a, $b )
 				$start = $first_event instanceof \WP_Post ? $first_event->dates->start_display : $user_date;
 			}
 			$end = $last_event instanceof \WP_Post ? $last_event->dates->start_display : $user_date;
+
+			// never let the start of the range be lower than the top bar date
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: f2ed83f6b6bb47d20b589dd2a59647c5</sub>